packages/container/package.json

```diff
@@ -25,7 +25,9 @@
   "devDependencies": {
     "@types/react-dom": "*",
     "@vitejs/plugin-react": "*",
+    "eslint": "*",
     "eslint-plugin-react": "*",
-    "eslint-plugin-react-hooks": "*"
+    "eslint-plugin-react-hooks": "*",
+    "vite": "*"
```

Comment:
can we have more specific version than *? it might break when there's a new major version